src/Crocodealer/Command/Sync.hs

```diff
@@ -0,0 +1,63 @@
+module Crocodealer.Command.Sync
+       ( sync
+       ) where
+
+import Network.HTTP.Simple (getResponseBody, getResponseStatus, httpBS, parseRequest)
+import Network.HTTP.Types.Status (Status (..))
+
+import Crocodealer.ColorTerminal (errorMessage, infoMessage, warningMessage)
+import Crocodealer.Core.GitHub (FileName (..), RepoName (..), UserName (..))
+
+data FileStatus
+    = UpToDate
+    | Outdated
+    | FileDoesNotExist
+
+sync :: UserName -> RepoName -> FileName -> [RepoName] -> IO ()
+sync userName templateRepoName fileName repos = do
+    templateFile <- fetchGitHubFile userName templateRepoName fileName
+    putTextLn $ unFileName fileName
+    forM_ repos $ \repo -> do
+        repoFile <- fetchGitHubFile userName repo fileName
+        let fileDiff = calculateDiff templateFile repoFile
+        printFileDiff userName repo fileDiff
+
+fetchGitHubFile :: UserName -> RepoName -> FileName -> IO (Maybe ByteString)
+fetchGitHubFile userName repoName fileName = do
+    let url = "https://raw.githubusercontent.com/" <> unUserName userName <> "/" <> unRepoName repoName <> "/master/" <> unFileName fileName
+    req <- parseRequest $ toString url
+    resp <- httpBS req
+
+    case getResponseStatus resp of
+        Status 200 _ -> pure $ Just $ getResponseBody resp
+        _            -> pure Nothing
+
+
+printFileDiff :: UserName -> RepoName -> FileStatus -> IO ()
+printFileDiff userName repoName fileStatus =
+    case fileStatus of
+        UpToDate ->
+            infoMessage $ user <> "/" <> repo <> ": " <> "Up-to-date"
+        Outdated ->
+            warningMessage $ user <> "/" <> repo <> ": " <> "Outdated"
+        FileDoesNotExist ->
+            errorMessage $ user <> "/" <> repo <> ": " <> "Doesn't have file"
+    where
+        user :: Text
+        user = unUserName userName
+
+        repo :: Text
+        repo = unRepoName repoName
```

Comment:
Instead of creating variables in `where` you can simply pattern-match on top-level:

```haskell
printFileDiff (UserName user) (RepoName repo) ...
```

Instead, you can create `userRepo = user <> "/" <> repo <> ": "` variable to avoid some code duplication.